.travis.yml

```diff
@@ -1,25 +1,69 @@
 sudo: required
 dist: trusty
 language: python
+compiler: gcc
 
 virtualenv:
   system_site_packages: true
 
 python:
   - 3.4
 
+env:
+  global:
+     # - QT_BASE=53
+     # - QT_BASE=54
+     # - QT_BASE=55
+     # - QT_BASE=56
+     - QT_BASE=57
+     #- GCC_ARM=/opt/gcc-arm-none-eabi/bin
+  matrix:
+     # - FLAVOR=ALL
+     - FLAVOR=DEFAULT
```

Comment:
Did you see my 2nd commit?  :)  I went back and forth between the names, and then meant to add a comment but forgot.